android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java

```diff
@@ -257,11 +257,18 @@ private void flattenAssets(final File sourceFile, final String dest)
   }
 
   public void onResume() {
-    this.activity.registerReceiver(
-        receiver,
-        new IntentFilter(DownloadService.NOTIFICATION),
-        RECEIVER_NOT_EXPORTED
-      );
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+      this.activity.registerReceiver(
+          receiver,
+          new IntentFilter(DownloadService.NOTIFICATION),
+          RECEIVER_NOT_EXPORTED
+        );
+    } else {
+      this.activity.registerReceiver(
+          receiver,
+          new IntentFilter(DownloadService.NOTIFICATION)
```

Comment:
what will happen for device under android 13, the receiver is missing

Reply:
this was working before

Reply:
@riderx The receiver was added in Android 13, so don't need it for Android < 12

Reply:
@riderx 

<img width="917" alt="image" src="https://github.com/Cap-go/capacitor-updater/assets/62795688/a07a8c14-4709-4fef-88e4-2d3de3689030">

Reply:
oh i didn't saw it was if else